tests/torch/test_utils.py

```diff
@@ -33,29 +36,22 @@ def compare_saved_model_state_and_current_model_state(model: nn.Module, model_st
         assert param.requires_grad == model_state.requires_grad_state[name]
 
 
-def randomly_change_model_state(module: nn.Module, compression_params_only: bool = False):
-    import random
+def change_model_state(module: nn.Module, trainable_paramaters: bool = True):
```

Comment:
Please remove `trainable_paramaters` param
```suggestion
def change_model_state(module: nn.Module):
```